docs/user-guide/cli-ftpplugin.md

```diff
@@ -39,45 +39,26 @@ Use one of the following methods to install or update the plug-in:
 
 ## Creating a user profile
 
-You create a zftp profile to avoid entering your connection details each time that you issue a command. You can create multiple profiles and switch between them as needed. Use one of the following methods to create a profile:
```

Comment:
Line 3, Line12: data sets should be two words
Line 16: Can you add "note" immediately after the three colons? (As is: **:::note**) I think I messed up the syntax here for the admonition. My bad.